tools/src/main/scala/com/nvidia/spark/rapids/tool/qualification/QualificationArgs.scala

```diff
@@ -42,10 +42,18 @@ Usage: java -cp rapids-4-spark-tools_2.12-<version>.jar:$SPARK_HOME/jars/*
           " eg: s3a://<BUCKET>/eventlog1 /path/to/eventlog2")
   val filterCriteria: ScallopOption[String] =
     opt[String](required = false,
-      descr = "Filter newest or oldest N eventlogs for processing." +
+      descr = "Filter newest or oldest N eventlogs based on timestamp of file for processing." +
```

Comment:
I think we should reword the description to have the first line state filter by filesystem timestamp, app timstamp, or app unique....   Then explain each one.